src/components/shared/Select.jsx

```diff
@@ -0,0 +1,17 @@
+import CONSTANT from "../../constants/constant";
+
+const { fieldTypes } = CONSTANT;
+
+function Select() {
+  return (
+    <select className="flex items-center w-[140px] h-7 mr-2 p-1 px-2 bg-light-grey text-center">
+      {fieldTypes.map(option => (
+        <option key={option} value={option}>
+          {option}
+        </option>
+      ))}
+    </select>
+  );
```

Comment:
```suggestion
function Select({ options }) {
  return (
    <select className="flex items-center w-[140px] h-7 mr-2 p-1 px-2 bg-light-grey text-center">
      {options.map(option => (
        <option key={option} value={option}>
          {option}
        </option>
      ))}
    </select>
  );
```

유진님! 현재 해당 Select 는 fieldTypes 만 렌더링 해주기에 범용적인 측면에서 조금 확장성이 떨어지지 않나 하는 걱정이 듭니다! 
만약 Select 컴포넌트가 prop을 받고 그 prop을 렌더링 해주는 방식은 어떠실까요? 
여기서 이제 propTypes로 options는 배열만 받을 수 있도록 해주면 좋을것 같아요!!

Reply:
네네 공감합니다. 저희 프로젝트에서 select가 쓰이는 곳이 단 한 곳이라 착각하고 말았네요!
constant를 prop으로 주도록 리팩토링하였습니다~!